tikv-configuration-file.md

```diff
@@ -996,6 +996,23 @@ Configuration items related to `rocksdb.defaultcf`
 + Default value: `"256GB"`
 + Unit: KB|MB|GB
 
+### `enable-compaction-guard`
+
++ Enables or disables the compaction guard, which is an optimization to split SST files at TiKV Region boundaries. This optimization can help reduce compaction I/O and allows TiKV to use larger SST file size (thus less SST files overall) and at the time efficiently clean up stale data when migrating Regions.
++ Default value: `true`
+
+### `compaction-guard-min-output-file-size`
+
++ The minimum SST file size when the compaction guard is enabled. This configuration prevents SST files from being too small when the compaction guard is enabled.
++ Default value: `"8MB"`
++ Unit: KB|MB|GB
+
+### `compaction-guard-max-output-file-size`
+
++ The maximum SST file size when the compaction guard is enabled. The configuration prevents SST files from being too large when the compaction guard is enabled. This configuration overrides `target-file-size-base` for the same column family.
++ Default value: `"128MB"`
++ Unit: KB|MB|GB
+
```

Comment:
Line 999 to Line 1014 LGTM
Please apply the same changes to Line 1114 - Line 1129.

Reply:
done